rustls/src/webpki/client_verifier.rs

```diff
@@ -42,6 +44,39 @@ impl ClientCertVerifierBuilder {
         }
     }
 
+    /// Clear the list of trust anchor subjects sent to the client as hints of acceptable trust
```

Comment:
Maybe "Clear the list of trust anchor subjects" is enough as a headline?

Reply:
Fixed.